requirements.txt

```diff
@@ -9,3 +9,4 @@ setuptools
 six
 typing_extensions
 dataclasses; python_version<"3.7"
+expecttest
```

Comment:
Can you submit another PR for this? We normally try to have one PR per issue

Reply:
OK